data/korath/korath missions.txt

```diff
@@ -577,3 +577,600 @@ mission "Efreti: Distress Call 8"
+
+			label harm
+			action
+				set "Korath [F05]: The Rainbow Feather: know"
+			`	"The Korath told me what you did: you brought a weapon to Far'en Lai, Captain." She looks deeply concerned. "That, alone, is terrible harm, and you should never do it again. They let you go because it is a planet of peace."`
+				to display
+					has "First Contact: Korath Exiles: pointed gun on Far'en Lai"
+				goto check
+			`	"Strangers landing on Far'en Lai is dangerous to the Korath, Captain <last>. If you make the wrong move, it can lead to their destruction. You did make a good impression when you agreed to bring them food. Any of the jobs at <origin>'s job board provide food to the Korath. Please lend all the help you can.`
+				to display
+					has "First Contact: Korath Exiles: agreed to bring food"
+				goto check
+			`	"Strangers landing on Far'en Lai is dangerous to the Korath, Captain <last>. If you make the wrong move, it can lead to their destruction. I'm glad you had the good sense not to do any real harm, but please stay away in the future.`
+				goto check
+
+			label know
+			action
+				set "Korath [F05]: The Rainbow Feather: know"
+			`	"We are allies of the Korath. Naturally, if a human somehow obtains a jump drive and wanders into Korath space, they ask us about it. Landing on Far'en Lai with a weapon does terrible harm. They let you go because it is a planet of peace."`
+				goto check
+
+			label feather
+			action
+				set "Korath [F05]: The Rainbow Feather: feather"
+			`	She removes her necklace and cups the rainbow feather tenderly in her hands. "This was a gift from a Korath woman who is my... close friend. Korath keep memories in their skin and pass knowledge by eating another's memories. The stronger the memory, the more colorful the feathers. This feather has the memories and emotions that she identifies with me.`
+			`	"If she dies in a raid, and another Korath eats this, I'll have her back, in a way. Another Korath that feels for me as she does, and remembers our times together. All Korath know the consequence of a rainbow feather. This is how they pass all knowledge; not through schooling, but through skin.`
+			`	"Think about that a moment. Years of memories flow in as if they were yesterday. Powerful emotions overwhelm you, and you don't understand why, because they aren't your own. You love someone who doesn't love you back because they never met you. Knowledge you couldn't comprehend before is intuitive, but it came at a high price. You'd lose all sense of time and self - unimaginable psychological pain to us is simply a fact of life to a Korath.`
+			`	"That, Captain, is the very core of their being. They never developed diplomacy because they didn't need it: a few feathers or the skin of the dead will end a conflict. That is, unless their terrible weapons annihilate the corpses, or if the enemies aren't Korath at all. We do what we can to mediate their troubles with humans, but we aren't always around."`
```

Comment:
```suggestion
			`	"That, Captain, is the very core of their being. They never developed diplomacy because they didn't need it: a few feathers or the skin of the dead may end a conflict. That is, unless their terrible weapons annihilate the corpses, or if the enemies aren't Korath at all. We do what we can to mediate their troubles with humans, but we aren't always around."`
```